<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <widget class="QMenu" name="Change_Theme">
     <property name="title">
      <string>Change Theme</string>
     </property>
     <addaction name="skin1"/>
     <addaction name="skin2"/>
     <addaction name="skin3"/>
    </widget>
    <addaction name="New_Game"/>
    <addaction name="Change_Theme"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="New_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="skin1">
   <property name="text">
    <string>原版</string>
   </property>
  </action>
  <action name="skin2">
   <property name="text">
    <string>DIO.VS.空条承太郎</string>
   </property>
  </action>
  <action name="skin3">
   <property name="text">
    <string>吉克.VS.利威</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
